plugins/main/public/components/overview/github/panel/config/drilldown-data-grid.tsx

```diff
@@ -32,6 +32,7 @@ export default function DrillDownDataGrid(props: tDrillDownDataGridProps) {
         }
         fetchData({
             query: searchBarProps.query,
+            filters: fetchFilters,
```

Comment:
**thought**: should the `fetchData` know the `fetchFilters` or they are other ones?